packages/core/content-type-builder/admin/src/components/AttributeOptions/AttributeOption/index.js

```diff
@@ -43,20 +34,34 @@ const AttributeOption = ({ type }) => {
     <OptionBoxWrapper padding={4} as="button" hasRadius type="button" onClick={handleClick}>
       <Flex>
         <AttributeIcon type={type} />
-        {/* TODO: Remove after the RTE Blocks Beta release (width) */}
+        {/* TODO: Remove after the RTE Blocks Stable release (width) */}
         <Box paddingLeft={4} width="100%">
-          {/* TODO: Remove after the RTE Blocks Beta release (justifyContent) */}
+          {/* TODO: Remove after the RTE Blocks Stable release (justifyContent) */}
           <Flex justifyContent="space-between">
             <Typography fontWeight="bold">
               {formatMessage({ id: getTrad(`attribute.${type}`), defaultMessage: type })}
             </Typography>
-            {/* Remove after the RTE Blocks Beta release */}
+            {/* Remove after the RTE Blocks Stable release */}
             {type === 'blocks' && (
-              <BetaTag>
-                <Typography textColor="secondary600" variant="sigma">
-                  BETA
-                </Typography>
-              </BetaTag>
+              <Flex grow={1} justifyContent="flex-end">
```

Comment:
I just checked and we'll want to have the New badge also for the Link field on v5. So it's not specific to blocks anymore. So I think we should make the code more generic:

```js
// top of the file
// TODO remove blocks from array on 4.16 release
const newAttributes = ['blocks']
const NewBadge = () => {}

// here
newAttributes.includes(type) && <NewBadge />
```